Backend/Api/Services/AccountService.cs

```diff
@@ -0,0 +1,27 @@
+﻿using Domain.Common;
+using Domain.User;
+using Microsoft.EntityFrameworkCore;
+
+
+
+namespace Api.Services
+{
+    public class AccountService
+    {
+
+        DomainDBContext ddbContext = new DomainDBContext();
```

Comment:
services should use IDataRepository,  not directly DbContexxt and should be placed in Logic project